spec/requests/authentication_controller_spec.rb

```diff
@@ -31,6 +31,12 @@
       get sign_in_path(redirect_path: "/hello-world")
       expect(AuthRequest.last.redirect_path).to eq("/hello-world")
     end
+
+    it "deletes old expired AuthRequests" do
```

Comment:
Are we going to have any kind of job for this too?
Like the thing we used to periodically clean up the RegistrationState and LoginState?

Reply:
Ah ha, just seen the commit message,
Ok sure.

I guess only thought here is, is there any peril in us having users who don't log in for a while who have crufy state hanging around?

Won't we need to set up sidekiq for other things too?

If not, then I think I agree this is a nice and simple approach.

Reply:
> I guess only thought here is, is there any peril in us having users who don't log in for a while who have crufy state hanging around?

That'll only happen if we have a bunch of auth requests created in a short period of time, and then no more for hours.  So I don't think it's much of a risk.

> Won't we need to set up sidekiq for other things too?

We'll probably end up needing to export attributes from here to bigquery, or something, which would be done best with sidekiq.  But we don't need it now and after the pain of getting the app running I'm not super keen on also getting sidekiq running...